src/NHibernate.Test/Async/NHSpecificTest/MultipleFieldsMappedToOneColumn/Fixture.cs

```diff
@@ -0,0 +1,125 @@
+﻿using System;
+using NHibernate.Cfg.MappingSchema;
+using NHibernate.Mapping.ByCode;
+using NUnit.Framework;
+
+namespace NHibernate.Test.Async.NHSpecificTest.MultipleFieldsMappedToOneColumn
+{
+	public class BadlyMappedEntity
+	{
+		public virtual long Id { get; set; }
+		public virtual long FirstValue { get; set; }
+		public virtual long SecondValue {get; set; }
+	}
+
+	public class CorrectlyMappedEntity
+	{
+		public virtual long Id { get; set; }
+		public virtual long FirstValue { get; set; }
+		public virtual long SecondValue {get; set; }
+	}
+
+	[TestFixture]
+	public class FixtureAsync : TestCaseMappingByCode
```

Comment:
The test should not be added in the `Async` subfolder. This folder contains only generated files, by the async generator. Move this in NHibernate.Test\NHSpecificTest, and rename it `Fixture`.

Reply:
done.